lib/elixir/test/elixir/gen_server_test.exs

```diff
@@ -51,7 +51,18 @@ defmodule GenServerTest do
   end
 
   test "call/3 exit messages" do
+    {:ok, pid} = GenServer.start_link(Stack, [:hello])
+
+    {:ok, stoped_pid} = GenServer.start_link(Stack, [:hello])
+    GenServer.stop(stoped_pid)
+
+    {:ok, killed_pid} = GenServer.start_link(Stack, [:hello], name: :stack)
+    GenServer.stop(:stack, :kill)
```

Comment:
@josevalim Thnx!
I forget about this behaviour.

But it still raises erlang error
```15:41:27.897 [error] GenServer :stack terminating
** (stop) :kill
Last message: []
State: [:hello]```
That I can't prevent anyhow.